src/Qir/Execution/Tools/QirTools.cs

```diff
@@ -29,27 +29,49 @@ public static class QirTools
             DirectoryInfo includeDirectory,
             DirectoryInfo executablesDirectory)
         {
-            using var qirContentStream = new MemoryStream();
-            if (!AssemblyLoader.LoadQirByteCode(qsharpDll, qirContentStream))
-            {
-                throw new ArgumentException("The given DLL does not contain QIR byte code.");
-            }
+            await Foo(qsharpDll, libraryDirectory, includeDirectory, executablesDirectory);
```

Comment:
This is just temporary for testing. It will be removed and this function restored before merging.